### DISCUSSION TITLE
Low resulation SBAS output

### DISCUSSION THREAD

**Discussion by sefayalvac** [gmtsar/gmtsar#541] [General]
Hello All,
Previous version GMTSAR analyzes were fine, but new version GMTSAR has very low resolution in SBAS grd outputs (please have a look in attachment png). Gray diagonal lines appear in grd file output. Any ideas on why this is and how I can fix it?
![Screenshot from 2022-11-07 20-56-15](https://user-images.githubusercontent.com/117675471/200383533-be230c12-3d2a-42e7-af4c-8e6229ea07dd.png)

**Comment by Xiaohua-Eric-Xu** [comment]
Could be due to your processing resolution. Not sure what resolution you are targeting, but generally if you specify 200 gaussian filter, the decimation should be around 8/2 in range/azimuth and the resulting geocoded product should be close to 50m resolution.
